M2/Macaulay2/packages/Oscillators.m2

```diff
@@ -0,0 +1,186 @@
+newPackage(
+        "Oscillators",
+        Version => "1.0", 
+        Date => "1 May 2025",
+        Authors => {
+            {
+                Name => "John Cobb", 
+                Email => "jdc0173@auburn.edu", 
+                HomePage => "https://johndcobb.github.io"
+                },
+            {
+                Name => "Hal Schenck", 
+                Email => "hks0015@auburn.edu",
+                HomePage => "http://webhome.auburn.edu/~hks0015"
+                },
+            {
+                Name => "Michael E. Stillman", 
+                Email => "mike@math.cornell.edu", 
+                HomePage => "http://www.math.cornell.edu/People/Faculty/stillman.html"
+                }
+            },
+        Headline => "code to analyze graph oscillators",
+        DebuggingMode => false,
+        AuxiliaryFiles => true,
+        PackageExports => {
+            "Graphs", 
+            "NumericalAlgebraicGeometry"
+            }
```

Comment:
There should be a `Keywords` option to `newPackage`, which is a list containing at least one string.  Ideally, keywords come from the list of existing headings on the [packages provided with Macaulay2](https://www.macaulay2.com/doc/Macaulay2/share/doc/Macaulay2/Macaulay2Doc/html/_packages_spprovided_spwith_sp__Macaulay2.html) doc page.